README.md

```diff
@@ -6,14 +6,14 @@ A repository that includes Hello World from popular languages added by our commu
 
 1. Fork the repository, and create a branch named `<language_to_add>`. In this example, we're adding **Java** to the list.
```

Comment:
Creating a branch isn't required. They're supposed to add the language directly into the `main` branch.

Reply:
That means: they should create a new branch in their repository, and get merged to the `main` branch. And that branch it's not going to be in this repository, it's going to be in their fork. 🙂